primaryHandler.go

```diff
@@ -167,19 +148,19 @@ func provideWebhookHandlers(in provideWebhookHandlersIn) (out provideWebhookHand
 		return
 	}
 
-	webhookConfig := in.WebhookConfigKey
-	webhookConfig.Logger = gokitLogger(in.Logger)
+	webhookConfig := in.WebhookConfig
+	webhookConfig.Logger = in.Logger
```

Comment:
Is this just because the `webhookConfig` type hasn't yet been updated to remove `go-kit`'s logger?

Reply:
Ah, yes this one sneak by me
I'll verify all of these have been accounted for

Reply:
I miss read your comment, if you're referring to `cannot use in.Logger (variable of type *zap.Logger) as "github.com/go-kit/log".Logger value in assignment: *zap.Logger does not implement "github.com/go-kit/log".Logger (wrong type for method Log)` then yes, that's what the https://github.com/xmidt-org/argus/pull/333 take care of